app/models/node.rb

```diff
@@ -214,7 +214,7 @@ def self.frequency(type, starting, ending)
     weeks = (ending.to_date - starting.to_date).to_i / 7.0
     Node.published.select(%i(created type))
       .where(type: type, created: starting.to_i..ending.to_i)
-      .count(:all) / weeks
+      .size(:all) / weeks
```

Comment:
`size(:all)`  is not a valid method ...lets remove `:all` here and leave it as `.size / weeks` we should get the correct value with that.

Reply:
Thank you for the advice. Changes made in the latest commit.